docs/pages/blog/docs-updates.md

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Our docs just got a major upgrade—here's what that means for you'
+description: Each of MUI's products now has its own dedicated documentation, making it easier than ever to find exactly what you need.
+date: 2022-04-06T00:00:00.000Z
+authors: ['danilo-leal']
+tags: ['News']
```

Comment:
If we ever https://www.notion.so/mui-org/Blog-SEO-structure-897c1ad742ec4274838a10f4bd6b3e3a put in place, it would then need this

```suggestion
tags: ['News', 'Product']
```

here.